tools/clang_tidy.py

```diff
@@ -262,13 +300,15 @@ def parse_options():
         action="store_true",
         help="Run clang tidy in parallel per-file (requires ninja to be installed).",
     )
+    parser.add_argument("-s", "--suppress-diagnostics", action="store_true",
+            help="Add NOLINT to suppress clang-tidy violations")
```

Comment:
I think this should fix it:
```suggestion
    parser.add_argument("-s", "--suppress-diagnostics", action="store_true",
                        help="Add NOLINT to suppress clang-tidy violations")
```